elk/utils_evaluation/utils_evaluation.py

```diff
@@ -18,6 +19,14 @@
 models_layer_num = default_config["models_layer_num"]
 
 
+def set_random_seed(seed):
+    """
+    Set the random seed
+    """
+    random.seed(seed)
+    np.random.seed(seed)
```

Comment:
Can you add `torch.manual_seed(seed)` here? We now use PyTorch randomness to initialize the probe weights